src/audio/kpb.c

```diff
@@ -472,53 +497,111 @@ static int kpb_reset(struct comp_dev *dev)
  */
 static int kpb_copy(struct comp_dev *dev)
 {
-	int ret = 0;
+	int ret = -EIO;
```

Comment:
Why EIO initially?

Reply:
Can you also add a little more detail in the commit message.

Reply:
@jajanusz corrected, @lgirdwood , commit message updated.